wayback-core/src/main/java/org/archive/wayback/archivalurl/FastArchivalUrlReplayParseEventHandler.java

```diff
@@ -277,6 +288,14 @@ private void handleOpenTagNode(ReplayParseContext context, TagNode tagNode)
 				emitHeadInsert(context, null, false);
 				// Don't return continue to further processing
 			}
+		} else if (tagName.equals("NOSCRIPT") && inHead) {
+			// NOSCRIPT under HEAD may contain elements that usually happen
+			// inside BODY. As long as they are inside NOSCRIPT, we let them go
+			// without inserting bodyInsert. Note this is NOSCRIPT under HEAD
+			// only. If <NOSCRIPT> happens before <HEAD>, non-HEAD elements in
```

Comment:
Do you see NOSCRIPT tags outside of the HEAD in practice? Mostly curious. Not sure I've come across that myself.

If it *does* happen in practice, should we make another issue to handle that case?